src/natives/screenshot/win/Program.cs

```diff
@@ -174,7 +174,7 @@ struct WindowBorderOnScreenFlags {
                     wi.rcClient.right -= 1;
                 }
             }
-            else if(browser == "ie") {
+            else if(browser == "iexplore") {
```

Comment:
There is no direct relation. `browser` variable actually holds a browser process name, and for IE it is `iexplore`. In previous version of `find-window` there was a special handling of IE, so it reported `"ie"` instead of right process name.